_dev/js/front/components/1_7/notification.component.js

```diff
@@ -55,6 +55,6 @@ export class NotificationComponent {
 
   showError(message) {
     this.notificationPaymentError.style.display = 'block';
-    this.notificationPaymentErrorText.textContent = message;
+    // this.notificationPaymentErrorText.textContent = message;
```

Comment:
why you uncomment this ? I commented it because the user doesn't care about Bad Request or Internal Server Error (related to the ticket 612)
```suggestion
    this.notificationPaymentErrorText.textContent = message;
```